troubleshoot/deployments/cloud-enterprise/rebuilding-broken-zookeeper-quorum.md

```diff
@@ -67,29 +67,46 @@ Perform the following steps on each host to back up the Zookeeper data directory
 
 ## Determine the Zookeeper leader [ece_determine_the_zookeeper_leader] 
 
-If a Zookeeper quorum is broken, you must establish the best Zookeeper leader to use for recovery before you start the recovery proces.
+If a Zookeeper quorum is broken, it's crucial to identify the best Zookeeper leader candidate to use for recovery before you start the recovery proces.
 
-The simplest way to check is using the [Zookeeper sync status](verify-zookeeper-sync-status.md) command.
+Collect the following Essential Information (A) and (B) from **all ECE director hosts that have ZK containers running**. This includes any recently created or decommissioned hosts.  Then reach out to [Elastic Support](/troubleshoot/index.md#contact-us).
```

Comment:
I'd like to see an explanation of why we need support to help with this - saw a couple of things listed in the linked issues around the risks of not identifying the right ZK leader that we might want to hint at here.